src/components/PropertyLookup.vue

```diff
@@ -34,7 +36,13 @@ export default Vue.extend( {
 	},
 	watch: {
 		async search( newSearchString: string ): Promise<void> {
-			this.searchResults = await this.$store.dispatch( 'searchProperties', newSearchString );
+			const searchResults = await this.$store.dispatch( 'searchProperties', newSearchString );
+			this.searchResults = searchResults.map( ( key: MenuItem & SearchResult ) => {
```

Comment:
Can we move this into the action?

Reply:
It needs i18n and that's not accessible in outside of the vue object. I can use it but it'll be pretty messy.

Reply:
Mh, I'll have a look. Having the i18n here, but the assigning the tag in the action should work and already make things a lot nicer